app/celery/research_mode_tasks.py

```diff
@@ -50,42 +43,16 @@ def send_email_response(reference, to):
     process_ses_results.apply_async([body], queue=QueueNames.RESEARCH_MODE)
 
 
-def make_request(notification_type, provider, data, headers):
-    api_call = "{}/notifications/{}/{}".format(current_app.config["API_HOST_NAME"], notification_type, provider)
-
-    try:
-        response = request("POST", api_call, headers=headers, data=data, timeout=60)
-        response.raise_for_status()
-    except RequestException as e:
-        api_error = HTTPError(e)
-        # TODO: Revert the following log to error level once the SMS research simulation has been fixed via:
-        #       https://trello.com/c/FTzY1HLJ/563-rework-fake-sms-callbacks-for-notifications-sent-using-test-keys
-        current_app.logger.info("API {} request on {} failed with {}".format("POST", api_call, api_error.response))
-        # TODO: Reinstate the error raise once the SMS research simulation has been fixed via:
-        #       https://trello.com/c/FTzY1HLJ/563-rework-fake-sms-callbacks-for-notifications-sent-using-test-keys
-        # raise api_error
-    finally:
-        current_app.logger.info("Mocked provider callback request finished")
-    return response.json()
-
+def aws_ses_callback(notification_id, to):
+    now = datetime.now()
+    timestamp = now.strftime("%m/%d/%Y %H:%M:%S")
```

Comment:
Is this an expected timestamp? I find it strange it's not in ISO 8601. Doesn't seem to match the documentation https://docs.aws.amazon.com/sns/latest/dg/sms_stats_cloudwatch.html

Reply:
Let's align with proper format..